src/components/Datatable/Pagination/PageButton.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import {
+  getColor,
+  getFontSize,
+  getFontWeight,
+  getLineHeight,
+  pxToRem,
+} from '../../../utils/helpers';
+import { Button } from '../../Button';
+import { ButtonVariants } from '../../Button/Button.enums';
+import { PageButtonProps } from './Pagination.types';
+
+const PageButton = styled(Button)
+  .attrs(() => ({
+    variant: ButtonVariants.text,
+  }))
+  .withConfig<PageButtonProps>({
+    shouldForwardProp: (prop) => prop !== 'isActive',
```

Comment:
Instead of `shouldForwardProp` you can use transient props notation (https://styled-components.com/docs/api#transient-props) so change `isActive` to `$isActive` and it won't be forwarded into the underlying component